dangerfile.ts

```diff
@@ -1,7 +1,18 @@
 'use strict';
 
-// eslint-disable-next-line node/no-missing-require
-const { danger, fail } = require('danger');
+// shamelessly "borrowed" from https://github.com/danger/danger-js/blob/master/source/danger.d.ts b/c we danger via npx
```

Comment:
add a comment to fix this in next major when we drop node 6 and can add a dep on danger rather than npx?

Reply:
we could also convert the meta files in a separate and just focus on the rules here

Reply:
I wondered if that was the reason, but couldn't find it in the danger changelogs :joy:

Reply:
I've broken this out into #368